spec/latest/core/core.jsonld

```diff
@@ -299,6 +302,27 @@
       "comment": "A collection holding references to a number of related resources.",
       "vs:term_status": "testing"
     },
+    {
+      "@id": "hydra:collection",
+      "@type": "hydra:Link",
+      "label": "collection",
+      "comment": "Allows to link multiple collections to an owning resource.",
+      "domain": "hydra:Collection",
+      "vs:term_status": "testing"
+    },
+    {
+      "@id": "hydra:manages",

```

Comment:
Do we have enough consensus about the manages block?

Reply:
Well, #41 is marked as resolved with `hydra:manages` as one of the solutions. While I think it indeed solves some problems, I've got personal issues with `hydra:collection` as I don't see any reason for making a special distinction between ordinary link and collection link (as this is how I understand `hydra:collection`)